receiptui/src/main/java/com/hyperwallet/android/ui/receipt/view/ReceiptDetailFragment.java

```diff
@@ -119,15 +119,18 @@ private void setTransactionView(@NonNull final Receipt receipt, @NonNull final V
         if (CREDIT.equals(receipt.getEntry())) {
             transactionAmount.setTextColor(transactionAmount.getContext()
                     .getResources().getColor(R.color.positiveColor));
-            transactionAmount.setText(CurrencyParser.getInstance(view.getContext()).formatCurrency(receipt.getCurrency(), receipt.getAmount()));
+            transactionAmount.setText(
+                    CurrencyParser.getInstance(view.getContext()).formatCurrency(receipt.getCurrency(),
+                            receipt.getAmount()));
             transactionTypeIcon.setTextColor(transactionTypeIcon.getContext()
                     .getResources().getColor(R.color.positiveColor));
             transactionTypeIcon.setText(transactionTypeIcon.getContext().getText(R.string.credit));
         } else if (DEBIT.equals(receipt.getEntry())) {
             transactionAmount.setTextColor(transactionAmount.getContext()
                     .getResources().getColor(R.color.negativeColor));
-            transactionAmount.setText(transactionAmount.getContext().getString(R.string.debit_sign_receipts,
-                    CurrencyParser.getInstance(view.getContext()).formatCurrency(receipt.getCurrency(), receipt.getAmount())));
+            transactionAmount.setText(transactionAmount.getContext().getString(R.string.debit_sign,
+                    CurrencyParser.getInstance(view.getContext()).formatCurrency(receipt.getCurrency(),
+                            receipt.getAmount())));
```

Comment:
I am assuming these are code formatting changes. Please confirm

Reply:
UI test failed, formatting is missing in UI SDK, but it's already working in App.